client/src/Components/postlist/postlist.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Post from './Posts.js';
+
+class PostList extends React.Component{
+    constructor(props){
+        super(props)
+        this.state ={
+            post:[{},{},{},{},{}]
```

Comment:
this would be a empty array and would be populated on component did mount via a api call.

Reply:
Ok bro I will try to resolve asap.